owncloudApp/src/main/java/com/owncloud/android/presentation/ui/files/operations/FileOperation.kt

```diff
@@ -19,11 +19,14 @@
  */
 package com.owncloud.android.presentation.ui.files.operations
 
+import android.accounts.Account
 import com.owncloud.android.domain.files.model.OCFile
 
 sealed class FileOperation {
```

Comment:
In this case, I think that it is accurate. So that, we have `FileOperation.CopyOperation`

Similar to `Color` idea. We expect `Color.YELLOW` and not `Colors.YELLOW`

What do you think?